### DISCUSSION TITLE
有人用破解？魔改？的PCL做整合包

### DISCUSSION THREAD

**Discussion by mysilent** [Meloong-Git/PCL#4472] [讨论]
![0e699ee5-9931-4691-aac1-9d10b6bb0a28](https://github.com/user-attachments/assets/187d8601-9164-46c8-889d-49b15b57b880)
![5b43e78b-2b73-4500-8111-5ea3be2490bc](https://github.com/user-attachments/assets/5cb5d232-0ab4-4700-8d2a-2ede5e487f74)
直接把exe传上来好像不太好
这个版本全皮肤都可以直接用
版权主页也进行了修改
如果管理需要可以给个邮箱我把他们的群号或者这个版本发过来看看

**Comment by allMagicNB** [comment]
需要群号以进一步调查

**Reply by mysilent** [reply]
怎么发出来

**Comment by MoYuan-CN** [comment]
郑重提醒：使用开源版 PCL（修改过的 PCL）制作整合包属于违反使用条例的行为

请将有关信息发送至 hexdragon@vip.qq.com 进行举报

感谢您的反馈

**Reply by mysilent** [reply]
好的

**Comment by WForst-Breeze** [comment]
鉴于该版本可以使用主题，因此可能并非是从开源版改的…… 传播此类 PCL 程序应当是绝对不允许的，尤其是用于服务器客户端这类可能导致广泛传播的用途

你可以将群号发送至楼上说的邮箱，或者让龙猫先来看看？

@LTCatt 打扰了 orz

**Reply by mysilent** [reply]
已经把群号发到上面的邮箱了

**Comment by LTCatt** [comment]
可以去群里小小提醒一下，对方没有恶意也不需要苛责 =。=

**Reply by allMagicNB** [reply]
他把赞助作者那块换成他的了。

**Reply by mysilent** [reply]
好 群号也发到了邮箱，龙猫大佬如果需要的话可以看看

**Reply by CrazyNutLz** [reply]
谢谢龙大，。。我改一下启动器只能登录我服务器

**Comment by sulingjiang** [comment]
Release 2.7.8?
正常不应该是 snapshot 吗(虽然拿到源码也呢改就是了)
minecraft ** 论坛可还行
怎么看怎么想反编译的版本
还有这赞助也被改了
~(感觉可能会出谣传)~

**Comment by IAmNotallMagicNB** [comment]
@CrazyNutLz
